code/modules/mob/living/carbon/human/species.dm

```diff
@@ -737,6 +738,8 @@
 					icon_num = 4
 				if(damage > (comparison*4))
 					icon_num = 5
+				if(H.hal_screwyhud == 5)
```

Comment:
I like this. We have another med that gives misleading "you are healthy" indicators (Aranesp), and it's one of the more entertaining things to dose someone with.